utils/litemap/src/map.rs

```diff
@@ -384,80 +342,6 @@ where
     }
 }
 
-impl<'a, K: 'a, V: 'a, S> LiteMap<K, V, S>
-where
-    K: Ord,
-    S: StoreIterableMut<'a, K, V> + StoreFromIterator<K, V>,
-{
-    /// Insert all elements from `other` into this `LiteMap`.
-    ///
-    /// If `other` contains keys that already exist in `self`, the values in `other` replace the
-    /// corresponding ones in `self`, and the rejected items from `self` are returned as a new
-    /// `LiteMap`. Otherwise, `None` is returned.
-    ///
-    /// The implementation of this function is optimized if `self` and `other` have no overlap.
-    ///
-    /// # Examples
-    ///
-    /// ```
-    /// use litemap::LiteMap;
-    ///
-    /// let mut map1 = LiteMap::new_vec();
-    /// map1.insert(1, "one");
-    /// map1.insert(2, "two");
-    ///
-    /// let mut map2 = LiteMap::new_vec();
-    /// map2.insert(2, "TWO");
-    /// map2.insert(4, "FOUR");
-    ///
-    /// let leftovers = map1.extend_from_litemap(map2);
-    ///
-    /// assert_eq!(map1.len(), 3);
-    /// assert_eq!(map1.get(&1), Some("one").as_ref());
-    /// assert_eq!(map1.get(&2), Some("TWO").as_ref());
-    /// assert_eq!(map1.get(&4), Some("FOUR").as_ref());
-    ///
-    /// let map3 = leftovers.expect("Duplicate keys");
-    /// assert_eq!(map3.len(), 1);
-    /// assert_eq!(map3.get(&2), Some("two").as_ref());
-    /// ```
-    pub fn extend_from_litemap(&mut self, other: Self) -> Option<Self> {
```

Comment:
Issue: I have found this method very useful. I see no reason to delete it.

Reply:
Added this method back.